README.md

```diff
@@ -29,42 +25,69 @@ RSpec.configure do |config|
   config.include JSONAPI::RSpec
 end
 ```
-
 ## Usage and documentation
 
-Available matchers:
+### Create a Helper to Convert the Response (Optional)
```

Comment:
I added this in as the first time I went to use the gem, it was very unclear what `document` was in the `expect(document).to have_meta` examples.  You're right it's opinionated so happy to remove it but but I do think the readme should explain what `document` is.  Ruby being duck typed, it could be anything :-)

Something like 
```ruby
# Example usage
let(:document) { JSON.parse(response.body, symbolize_names:  false) }
```
above the available matchers might help?